### ISSUE TITLE
case sensitive

### ISSUE THREAD

**Issue by Aravind03** [sitecorerick/constellation-sitecore9#15]
Hi @sitecorerick ,

We have identified that the redirection is not working for the below case. 

Example : Assuming we have a page www.abc.com/Help which needs to be redirected to www.abc.com/contact-us.  If the user browse for www.abc.com/help (with small 'h'), then it's redirecting to custom 404 page.

Any help is appreciated.

**Comment by sitecorerick** [comment]
This is because IIS is not case-sensitive. You need to install an IIS
UrlRewrite rule to force all URLs to lowercase to prevent this from
happening. It's standard best practice for SEO to ensure all URLs are
lowercase.

**Comment by Aravind03** [comment]
@sitecorerick  - Thanks for the information.

Is it possible to download the source code for the project(Constellation.Feature.Redirects), make some code changes and build a custom package out of it which can be uploaded to sitecore.

If yes, can you please provide the steps to be performed.

**Comment by sitecorerick** [comment]
Yes, you can do that. The location of the repository is in the NuGet
package description.
